File: 5-matematica/1555.py
q = int(input())
for p in range(q):
    e = str(input()).split()
    x = int(e[0])
    y = int(e[1])
    r = ((3 * x) ** 2) + (y ** 2)
    b = 2 * (x ** 2) + ((5 * y) ** 2)
    c = -100 * x + (y ** 3)
    if r > b:
        if r > c: print('Rafael ganhou')
        else: print('Carlos ganhou')
    else:
        if b > c: print('Beto ganhou')
        else: print('Carlos ganhou')

File: 5-matematica/1555.c
#include <stdio.h>
#include <math.h>

int main(void) {
    int n, i, x, y, r, b, c;

    scanf("%d", &n);
    for (i = 0; i < n; ++i) {
        scanf("%d %d", &x, &y);
        r = pow((3 * x), 2) + pow(y, 2);
        b = 2 * pow(x, 2) + pow((5 * y), 2);
        c = -100 * x + pow(y, 3);
        if (r > b) {
            if (r > c) printf("Rafael ganhou\n");
            else printf("Carlos ganhou\n");
        }
        else {
            if (b > c) printf("Beto ganhou\n");
            else printf("Carlos ganhou\n");
        }
    }
    return 0;
}


